README.md

```diff
@@ -1,16 +1,31 @@
-# AI RHDH Installer
+# Red Hat Developer Hub Installer
 
-### Installation
+This helm chart installs and configures the following operators:
```

Comment:
I think we should also include the RHDH operator in this list as well, since developer hub is going to be deployed using the operator instead of the helm chart.

@thepetk Would it make sense that we include this in this PR? Or I can either add this under my [PR](https://github.com/redhat-ai-dev/ai-rhdh-installer/pull/3) after this one is merged.

Reply:
Yeah sure, feel free to add it here :)

Reply:
Okay I'll add it under https://github.com/redhat-ai-dev/ai-rhdh-installer/pull/3 after this is merged.